frontend/src/metabase/parameters/components/ParameterWidget/ParameterWidget.unit.spec.tsx

```diff
@@ -60,7 +60,11 @@ describe("ParameterWidget", () => {
     userEvent.type(textInput, "{backspace}".repeat(text.length));
     expect(screen.getByRole("button", { name: "Add filter" })).toBeDisabled();
 
-    userEvent.click(screen.getByRole("button", { name: "Add filter" }));
+    userEvent.click(
+      screen.getByRole("button", { name: "Add filter" }),
+      undefined,
+      { skipPointerEventsCheck: true },
```

Comment:
could you please explain why it is required?

Reply:
Now, as we're using the Mantine button, it implements `pointer-events: none` for the `disabled` button for some reason. This means the testing lib starts complaining that it couldn't click the button that has this property. 

So instead of changing the test, which I think is valid (it's okay to try to click a disabled button), I simply "allowed" it to perform a click.